packages/chain-adapters/src/utxo/types.ts

```diff
@@ -74,6 +74,8 @@ export type FeeData = {
 }
 
 export type BuildTxInput = {
+  // The from address UTXOs should be filtered by
+  // Note, this voids all privacy guarantees of UTXO-based chains
   from?: string
```

Comment:
yeah, I wanted to write `only for savers atm` but feels like this may changed - selecting UTXOs isn't an exotic feature, it's actually a relatively common use-case we may run into for other features

Reply:
utxo selection isn't exotic by any means, but selection strategy using a `from` address isn't common or useful outside of savers as far as i'm aware. not big deal, just thought it could add some useful context